trp_use_case/model/use_case.py

```diff
@@ -59,18 +59,19 @@ def _get_use_case_hours(self):
             values = {
                 'hours': 0.0,
                 'hours_optional': 0.0,
+                'hours_nonoptional': 0.0,
             }
             for workload in use_case.workload_ids:
                 values['hours'] += workload.hours
                 if workload.optional:
                     values['hours_optional'] += workload.hours
                     values['hours_nonoptional'] = (
```

Comment:
this was also out of the loop: https://github.com/Therp/Therp-Addons/commit/9c654f4eb21d482597fdf37a0ef27c7c0f3c8ff4#diff-2a86637de0625d93b7be3df3f4de7d0bL67